examples/MvcWeb/TheraLang/ProjectStatusId.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace MvcWeb.TheraLang
+{
+    public enum ProjectStatusId
```

Comment:
this is not ProjectStatusId it should be
```suggestion
    public enum ProjectStatus
```
and I do not see where you used it? why haven't you applied it to the code above? please do that as I already commented